src/writing-your-own-application/configuration.md

```diff
@@ -1 +1,25 @@
 # Configuration
+
+The [esp-config] crate provides a way how to manage configuration settings.
+
+
+## Usage
+
+The full list of available options could be found in `out` dir with the format `{prefix}_config_table.md`, for example `esp_hal_config_table.md` when changed options are `esp-hal`-related.
+
+If you want to place a function in your code in the RAM (ESP_HAL_CONFIG_PLACE_SPI_DRIVER_IN_RAM), which is set to false as default, you need to create an environment variable with the same name and modify its value:
```

Comment:
We should say something like `For example: If you want ....`

Otherwise, it's a confusing change of topics